### ISSUE TITLE
Android build arguments are different since Ren'py 7.4.9

### ISSUE THREAD

**Issue by SergioAlonsoJr** [kobaltcore/renconstruct#11]
The Android build cli no longer uses the  "assembleRelease" argument

Found this in renpy github android.py file

<img width="852" alt="image" src="https://user-images.githubusercontent.com/4298411/168448532-6e06c668-6631-46ad-993b-3c3e427557a4.png">

If you want to build Android App Bundle format, use the --bundle argument. 
If you want to build Universal APK, remove the --bundle argument.

[Ren'py 7.4.9 Change log](https://www.renpy.org/doc/html/changelog.html#android)

Still, there are other obstacles to solve, but this is one step ahead.

**Comment by kobaltcore** [comment]
Ah, good catch. I'll audit the last few versions for other changes and then I'll add a version guard to account for this change in API on Ren'Py's side.

**Comment by SergioAlonsoJr** [comment]
One extra update:

I had to make at least one android build using the interface before the command line would work correctly.
I think there are some configuration steps that are not being done by renconstruct.

**Comment by kobaltcore** [comment]
Can not confirm this at the moment, it's building android distributions fine on my end right now.
I'll take a deeper look tomorrow.

**Comment by SergioAlonsoJr** [comment]
> Can not confirm this at the moment, it's building android distributions fine on my end right now. I'll take a deeper look tomorrow.

I think the error I was facing was related to the Dropbox trying to sync the android temporary files.
I moved the build process outside of Dropbox and it seems to be working fine now.

**Comment by kobaltcore** [comment]
Yeah, file syncing stuff like that seems to quickly break down with file-heavy operations such as these (where Ren'Py creates thousands upon thousands of often-tiny files). I've seen issues related to that on most of these services, including OneDrive, iCloud, Dropbox and Google Drive, so it would be recommended to move at least the cache locations out of the purview of these programs, so that they have no chance to mess with the internal state.
